lib/x11utils.pm

```diff
@@ -182,12 +218,25 @@ sub handle_logout {
     assert_and_click 'logout-button';                                  # press logout
 }
 
+=head2 handle_relogin
+
+ handle_relogin();
+
+First logs out and the log in via C<handle_logout> and C<handle_login>
+
+=cut
 sub handle_relogin {
     handle_logout;
     handle_login;
 }
 
-# Handle the case when user is not selected, on gnome
+=head2 select_user_gnome
+
+ select_user_gnome($myuser);
+
+Handle the case when user is not selected in login screen, on gnome
```

Comment:
```suggestion
Handle the case when user is not selected in login screen, on gnome.
C<$myuser> specifies the username to switch to.
If not set, it will default to C<$username>.
```